cvat/apps/iam/serializers.py

```diff
@@ -56,7 +57,7 @@ def save(self, request):
         if "password1" in self.cleaned_data:
             try:
                 adapter.clean_password(self.cleaned_data['password1'], user=user)
-            except ValidationError as exc:
```

Comment:
Wrong exception class used, it caused 500 server error because of unhandled exception